capture_gui/__init__.py

```diff
@@ -16,10 +17,12 @@ def main(show=True):
     """
     # get main maya window to parent widget to
     parent = lib.get_maya_main_window()
+    instance = parent.findChild(QtWidgets.QWidget, "CaptureGUI")
```

Comment:
Would it make more sense to change `"CaptureGUI"` to `app.App.object_name` so they are always in sync?